src/sources/soundsourceproxy.cpp

```diff
@@ -422,97 +426,119 @@ void SoundSourceProxy::updateTrackFromSource(
         return; // abort
     }
     if (!m_pSoundSource) {
-        kLogger.warning() << "Unable to parse tags from unsupported file type"
-                 << getUrl().toString();
+        kLogger.warning()
+                << "Unable to update track from unsupported file type"
+                << getUrl().toString();
         return; // abort
     }
 
-    // Use the existing trackMetadata as default values. Otherwise
-    // existing values in the library will be overwritten with
-    // empty values if the corresponding file tags are missing.
-    // Depending on the file type some kind of tags might even
-    // not be supported at all and those would get lost!
+    // Initialize or update the file/format as reported by the associated
```

Comment:
Does this comment add any value? Maybe it is more interesting who else set the type. It looks as this is the only place though .